src/main/java/com/synopsys/integration/alert/channel/jira/server/descriptor/JiraServerDistributionUIConfig.java

```diff
@@ -43,7 +43,7 @@
     private static final String LABEL_RESOLVE_WORKFLOW_TRANSITION = "Resolve Transition";
     private static final String LABEL_OPEN_WORKFLOW_TRANSITION = "Re-open Transition";
     private static final String DESCRIPTION_ADD_COMMENTS = "If true, this will add comments to the Jira ticket with data describing the latest change.";
-    private static final String DESCRIPTION_ISSUE_CREATOR = "The email of the Jira Server user to assign as the issue creator field of the Jira issue.";
+    private static final String DESCRIPTION_ISSUE_CREATOR = "The username of the Jira Server user to assign as the issue creator field of the Jira issue.";
```

Comment:
This was email before due to a copy paste error from the Jira Cloud implementation. Changed to a single word.